src/main/java/games/strategy/triplea/ai/proAI/ProTechAI.java

```diff
@@ -122,7 +121,6 @@ private static float getStrengthOfPotentialAttackers(final Territory location, f
       final Match<Unit> aTransport = Match.allOf(Matches.UnitIsSea, Matches.UnitIsTransport, Matches.UnitCanMove);
       final List<Territory> eFTerrs = findUnitTerr(data, enemyPlane);
       int maxFighterDistance = 0;
-      int maxBomberDistance = 0;
```

Comment:
This variable appears to no longer have a purpose.  After the adjustment below on old lines 138-142, it is never used again.  So I simply removed it.